public/static/js/subscriptions.js

```diff
@@ -62,29 +86,13 @@ function subCtrl($scope, $mdDialog) {
     // labels should complete the following sentence fragments:
     // 'then notify by ...'
     // 'when ...'
-    $scope.subscription_methods = [
-        {
-            value: SUBSCRIPTION_EMAIL,
-            label: "sending an email",
-        },
-        {
-            value: SUBSCRIPTION_SLACK,
-            label: "sending a slack message",
-        },
-        {
-            value: SUBSCRIPTION_JIRA_COMMENT,
-            label: "making a comment on a JIRA issue",
-        },
-        {
-            value: SUBSCRIPTION_JIRA_ISSUE,
-            label: "making a JIRA issue",
-        },
-        {
-            value: SUBSCRIPTION_EVERGREEN_WEBHOOK,
-            label: "posting to an external server",
-        },
-        // Github status api is deliberately omitted here
-    ];
+
+    $scope.subscription_methods = DEFAULT_SUBSCRIPTION_METHODS;
+    if ($scope.c.omit) {
+      $scope.subscription_methods = _($scope.subscription_methods).filter(function(method){
```

Comment:
I think you can compress this expression some more, `_(DEFAULT_SUBSCRIPTION_METHODS)...` if you assign omit to the empty object if omitMethod is undefined in `subscriberPromise`

Reply:
I feel like that just moves a bit of verbosity from one place to another. I think this way is slightly cleaner